requests/packages/chardet/chardetect.py

```diff
@@ -17,7 +17,6 @@
 
 import argparse
 import sys
-from io import open
```

Comment:
Can you remove the changes from this file please? This is a third-party package we vendor unchanged.